examples/asr/transcribe_speech.py

```diff
@@ -154,6 +154,9 @@ class TranscriptionConfig:
 def main(cfg: TranscriptionConfig) -> TranscriptionConfig:
     logging.info(f'Hydra config: {OmegaConf.to_yaml(cfg)}')
 
+    for key in cfg:
```

Comment:
Is this necessary?
When configs with None values has their values as string?

Reply:
asr_evaluator will call the script with subprocess.run( ) which will make it to be "None".  this is to avoid the case like this